src/parsers/manifest/dash/parse_adaptation_sets.ts

```diff
@@ -141,15 +143,19 @@ function hasSignLanguageInterpretation(
  * Contruct Adaptation ID from the information we have.
  * @param {Object} adaptation
  * @param {Array.<Object>} representations
+ * @param {Array.<Object>} representations
  * @param {Object} infos
+ * @param {Boolean} isTrickModeTrack
  * @returns {string}
  */
 function getAdaptationID(
   adaptation : IAdaptationSetIntermediateRepresentation,
+  representations : IParsedRepresentation[],
   infos : { isClosedCaption : boolean | undefined;
             isAudioDescription : boolean | undefined;
             isSignInterpreted : boolean | undefined;
-            type : string; }
+            type : string; },
+  isTrickModeTrack: boolean
```

Comment:
Why not adding that to the `infos` part? Seems to make sense to put it there by reading that function's signature